pkgs/applications/virtualization/nvidia-docker/libnvc.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv, lib, fetchFromGitHub, fetchurl, libelf, libcap, libseccomp
+, nvidia_x11 }:
+
+with lib; let
+
+  persistenced = nvidia_x11.passthru.persistenced;
+
+in stdenv.mkDerivation rec {
+  name = "libnvidia-container-${version}";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "NVIDIA";
+    repo = "libnvidia-container";
+    rev = "v1.0.0";
+    sha256 = "1ws6mfsbgxhzlb5w1r8qqg2arvxkr21n59i4cqsyz3h5jsqsflbw";
+  };
+
+  nvidia-modprobe = fetchurl {
+    url = "https://github.com/NVIDIA/nvidia-modprobe/archive/396.51.tar.gz";
+    sha256 = "082b1jvkg1dij6l7yvmi7vi1fdbmp7jw4snpkw76ggl4lcvn9g15";
+  };
+
+  patches = [ ./libnvc-ldconfig-and-path-fixes.patch ];
```

Comment:
And what's this patch for? A comment would be nice.

Reply:
I've added some comments here. It's possible there might be a way to make ldconfig work directly, by specifying the search path, the cache file path etc. directly making it writable etc. But that is still two patches, and it has to find exactly these included nvidia_x11 libraries in this version anyway. And also the PATH would still need a patch or wrapping.